websites/D/Daily Mail/presence.ts

```diff
@@ -0,0 +1,68 @@
+import { findPage } from "./findPage";
+import { handleNewsPage } from "./newsPage";
+
+const presence = new Presence({
+	clientId: "1217153856665026580",
+});
+
+export const enum Assets {
+	Logo = "https://i.imgur.com/glm7on6.png",
+	ExclIco = "https://i.imgur.com/FitEOZS.png",
+	PlayIco = "https://i.imgur.com/rmA1FMA.jpeg",
+	PauseIco = "https://i.imgur.com/DFFBv1a.png",
```

Comment:
Why not use the existing play pause icons?

Reply:
because I found out they existed after I pushed this